### PR TITLE
[GFX11][tests] New target additional list file changes

### PR DESCRIPTION
**Pull request by muralinr**
target addition list file changes

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -38,6 +38,7 @@ option( MIOPEN_TEST_GFX90A "Test on gfx90a" OFF )
 option( MIOPEN_TEST_GFX900 "Test on Vega10 (gfx900)" OFF )
 option( MIOPEN_TEST_GFX906 "Test on Vega20 (gfx906)" OFF )
 option( MIOPEN_TEST_GFX103X "Test on Navi21/22 (gfx1030/31)" OFF )
+option( MIOPEN_TEST_GFX110X "Test on Navi31/32 (gfx1100/01)" OFF )
```
Comment (atamazov) [inline test/CMakeLists.txt]: Typo
```suggestion
option( MIOPEN_TEST_GFX110X "Test on Navi31/32 (gfx1100/02)" OFF )
```

**Review by atamazov — CHANGES_REQUESTED** [review]

**Comment by atamazov** [comment]
@muralinr Please prepend `[GFX11][tests]` to the title of this PR.

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -100,6 +101,10 @@ if(NOT (MIOPEN_TEST_GFX900 OR MIOPEN_TEST_GFX906 OR MIOPEN_TEST_GFX908 OR MIOPEN
             set(MIOPEN_TEST_GPU_DETECTION_FAILED TRUE)
         elseif(ROCMINFO_OUTPUT MATCHES "gfx1030")
             set(MIOPEN_TEST_GFX103X ON)
+        elseif(ROCMINFO_OUTPUT MATCHES "gfx1100")
```
Comment (atamazov) [inline test/CMakeLists.txt:104]: @muralinr Please append `OR MIOPEN_TEST_GFX110X` to line 80.

**Comment by atamazov** [comment]
@muralinr Please insert
```
message(STATUS "MIOPEN_TEST_GFX110X ${MIOPEN_TEST_GFX110X}")
```
just after this:
```
message(STATUS "MIOPEN_TEST_GFX103X ${MIOPEN_TEST_GFX103X}")
```

**Comment by atamazov** [comment]
@junliume @muralinr [Important] No **_custom tests_** are enabled for the new target right now. Should we enable (approximately) the same custom tests as we have for GFX103X? I can make a PR for this.

**Comment by muralinr** [comment]
> @junliume @muralinr [Important] No **_custom tests_** are enabled for the new target right now. Should we enable (approximately) the same custom tests as we have for GFX103X? I can make a PR for this.

Yes Artem. Can you please make a PR?

**Comment by atamazov** [comment]
@muralinr Sure. I can do that after this PR merged, or I can consolidate changes from here + my changes into a new PR. If you prefer the latter way, please close this one.

**Review by atamazov — APPROVED** [review]
LGTM!

**Comment by muralinr** [comment]
> @muralinr Sure. I can do that after this PR merged, or I can consolidate changes from here + my changes into a new PR. If you prefer the latter way, please close this one.

Sure Artem. Either way is fine. I will close this PR if you put these changes in your PR. Thank you.

**Comment by atamazov** [comment]
@muralinr #1925 created, this PR can be closed now.

Can you please #1925 it on the appropriate target and make sure it passes our CI? Thanks! In case of any problem please let me know or fix it yourself. In order to make changes in my PR, please accept the invitation to my fork of MIOpen, https://github.com/atamazov/MIOpen (I've sent the invitation several mins ago). Then you'll be able to `git remote add atamazov_miopen https://github.com/atamazov/MIOpen`, pull branches from there, edit and push them back as usual.

[Informative] If you're curious, I am unable to create branches in this repo due to lack of privileges.

File: test/CMakeLists.txt (hunk 2 above)
Comment (atamazov) [inline test/CMakeLists.txt:104]: +gfx1101?

**Pull request closed without merging**